src/software/world/robot.cpp

```diff
@@ -170,3 +171,31 @@ Polygon Robot::dribblerArea() const
              vector_to_front.perpendicular().normalize(width),
          bottom_left_position + vector_to_front.perpendicular().normalize(width)});
 }
+
+
+Duration Robot::getTimeToOrientation(const Angle &desired_orientation) const
+{
+    double dist = orientation().minDiff(desired_orientation).toRadians();
```

Comment:
so we're implicitly finding positive values? Can we have a test where desired orientation is less than the current orientation?

Reply:
Added a new test where the destination angle is less than the starting angle